migrations/000_db_create_tables.php

```diff
@@ -26,6 +29,17 @@
             );
         ',
     ];
+
+    if ($use_multitenant) {
```

Comment:
Database changes should be added to a new migration file, with the three digit number at the beginning of the filename incrementing by 1 each time.  In this case, it would be something like `004_add_multitenant`.